examples/feature_extraction/peft_lora_embedding_semantic_search.py

```diff
@@ -207,9 +207,18 @@ def get_loss(cosine_score, labels):
 
 def main():
     args = parse_args()
+    
+    # initialize accelerator
```

Comment:
I think the comment is not necessary, as it is quite clear what happens here.

Regarding the initialization arguments, maybe they can be cleaned up a bit. WDYT about something like:

```python
accelerator_kwargs = {"gradient_accumulation_steps": args.gradient_accumulation_steps}
if args.with_tracking:
    accelerator_kwargs["log_with"] = args.report_to
    accelerator_kwargs["project_dir"] = args.output_dir
accelerator = Accelerator(**accelerator_kwargs)
```

Reply:
update ! Thanks !